packages/core/src/IconButton.js

```diff
@@ -26,10 +25,6 @@ const IconButton = ({ name, size, color, ...props }) => (
 )
 
 IconButton.displayName = 'IconButton'
-
-IconButton.defaultProps = {
-  theme: theme
-}
 IconButton.isIconButton = true
```

Comment:
What is this property used for and can we remove it, just seems unnecessary or potentially hiding a problem.

Reply:
this property is used in `IconField` to check the type of component (icon or button) that is passed in.